beard/clustering/tests/test_wrappers.py

```diff
@@ -31,6 +34,14 @@ def test_scipy_hierarchical_clustering():
     random_state = check_random_state(42)
     X, y = make_blobs(centers=4, shuffle=False, random_state=random_state)
 
+    # Unsupervised clustering
```

Comment:
This tests: `test_scipy_hierarchical_clustering` and `test_scipy_hierarchical_clustering_semi_supervised` contain a lot of use cases. It might be worth to split it. You can use `pytest.fixture` to get the same `X` and `y` for every test case.